.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: dtolnay/rust-toolchain@v1
         with:
-          toolchain: stable
+          toolchain: nightly
```

Comment:
Is nightly really still needed here?

Reply:
Oh, huh: looks like it's not for `clippy` (this part). I lean towards keeping it, to line it up with the rustfmt and check builds that both use nightly.

Reply:
I hadn't looked, in that case, i agree.